business-central/pom.xml

```diff
@@ -80,121 +79,84 @@
-                <argument>-t</argument>
-                <argument>${docker.kie.repository}/business-central-wildfly19:${docker.build.kie.tag}</argument>
-                <argument>${project.basedir}/src/main/docker/wildfly</argument>
-              </arguments>
-            </configuration>
-          </plugin>
-       <!--
         <plugins>
             <plugin>
-                <groupId>io.fabric8</groupId>
-                <artifactId>docker-maven-plugin</artifactId>
-                <configuration>
-                    <images>
-                        <image>
-                            <name>${docker.kie.repository}/business-central-wildfly19:${docker.build.kie.tag}</name>
-                            <alias>business-central-wildfly19-${docker.build.kie.tag}-h2</alias>
-                            <build>
-                                <dockerFileDir>${project.build.outputDirectory}/wildfly</dockerFileDir>
-                                <assembly>
-                                    <descriptor>${pom.basedir}/src/main/assembly/assembly-business-central-wildfly19.xml</descriptor>
-                                </assembly>
-                            </build>
-                            <run>
-                                <namingStrategy>alias</namingStrategy>
-                                <env>
-                                    <KIE_CONNECTION_URL>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</KIE_CONNECTION_URL>
-                                    <KIE_CONNECTION_DRIVER>h2</KIE_CONNECTION_DRIVER>
-                                    <KIE_CONNECTION_USER>sa</KIE_CONNECTION_USER>
-                                    <KIE_CONNECTION_PASSWORD>sa</KIE_CONNECTION_PASSWORD>
-                                    <KIE_CONNECTION_DATABASE>kie</KIE_CONNECTION_DATABASE>
-                                </env>
-                                <ports>
-                                    <port>kiedroolswb.wf19.http.port:8080</port>
-                                </ports>
-                                <wait>
-                                    <http>
-                                        <url>http://localhost:${kiedroolswb.wf19.http.port}/business-central</url>
-                                        <method>GET</method>
-                                    </http>
-                                    <time>${docker.run.timeout}</time>
-                                </wait>
-                                <log>
-                                    <prefix>business-central-wildfly19</prefix>
-                                    <date>ISO8601</date>
-                                    <color>blue</color>
-                                </log>
-                            </run>
-                        </image>
-                    </images>
-                </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
i used at the end maven-dependency-plugin instead of assembly plugin as the descriptor was a little bit limited because docker-maven-plugin probably just used just "similar" assembly descriptor.